charts/helm_lib/templates/_module_image.tpl

```diff
@@ -4,6 +4,9 @@
   {{- $context := index . 0 }} {{- /* Template context with .Values, .Chart, etc */ -}}
   {{- $containerName := index . 1 | trimAll "\"" }} {{- /* Container name */ -}}
   {{- $moduleName := $context.Chart.Name | replace "-" "_" | camelcase | untitle }}
+  {{- if ge (len .) 3 }}
+  {{- $moduleName = index . 2 | trimAll "\"" | replace "-" "_" | camelcase | untitle }} {{- /* Optional module name */ -}}
+  {{- end }}
```

Comment:
We have function helm_lib_module_camelcase_name in helm_lib. I think we should modify this function and use it